website/docs/r/application_epg.html.markdown

```diff
@@ -49,7 +50,7 @@ resource "aci_application_epg" "fooapplication_epg" {
 * `prio` - (Optional) QoS priority class id. Allowed values are "unspecified", "level1", "level2", "level3", "level4","level5" and "level6". By default the value is inherited from the parent application profile.
 * `shutdown` - (Optional) Shutdown for object application epg. Allowed values are "yes" and "no". Default is "no".
 
-* `relation_fv_rs_bd` - (Required) Relation to Bridge domain associated with EPG (Point to class fvBD). Cardinality - N_TO_ONE. Type - String.
+* `relation_fv_rs_bd` - (Optional) Relation to Bridge domain associated with EPG (Point to class fvBD). Cardinality - N_TO_ONE. Type - String.
```

Comment:
I am conflicted about the BD relationship. It is not mandatory in the API but the APIC UI made it required at some point. There is no real use case to create an EPG without a BD relationship but I don't feel making it mandatory in the go file is a good idea as it might break someones existing configuration. That is why we have kept it mandatory in the documentation but optional in the code so people that are new to this resource understand they should configure it to have a functional EPG but we leave the option for people not to configure it this way.

Any specific reason you want to change this documentation?